README.adoc

```diff
@@ -57,12 +57,18 @@ Try to use at least version `0.11.0` as we haven't tested another version.
 Assuming you have Go installed and configured (have `$GOPATH` setup) here is
 how to build.
 
-Check out the code
+Check out the code. On Linux or Mac:
 
 ----
 $ git clone https://github.com/almighty/almighty-core $GOPATH/src/github.com/almighty/almighty-core
 ----
 
+On Windows PowerShell:
+
+----
+$ git clone https://github.com/almighty/almighty-core $env:GOPATH\src\github.com\almighty\almighty-core
+----
```

Comment:
created https://github.com/almighty/almighty-core/pull/76 as alternative - it is enough imo to have the makefile windows compatible. To repeat Windows commands for each of its shell syntaxes (powershell and cmd.exe) in addition to posix is not worth the hassle imo.